src/config/configure.in

```diff
@@ -59,7 +59,7 @@ m4_include([config/hypre_blas_macros.m4])
 m4_include([config/hypre_lapack_macros.m4])
 m4_include([config/hypre_macros_misc.m4])
 
-AC_PREREQ(2.59)
+AC_PREREQ([2.69])
```

Comment:
Why 2.69 here? Don't we now want to require 2.71?

Reply:
I want to allow users to build the `configure` script with `autoconf` 2.69 in the event they have issues with this update.  I don't think there will be issues, but it's a fall back that helped to ease concerns some of our users had about this update.  I've tested everything to make sure 2.69 still produces a working `configure` script.

Reply:
So, 2.69 is the minimum version we support from now on?

Reply:
(which is, 11-year old, BTW)

Reply:
> So, 2.69 is the minimum version we support from now on?

Yes, sort of.  I don't think we'll have to worry about this.  I am not aware of any users who have ever built their own `configure` script from our `configure.in`.  We don't want them to do that.  I just want to have a fall back for a short time, just in case somebody has an issue.  Developers will still be required to use 2.71.  Note that the numbers we had in the `AC_PREREQ` statements before were all over the map, but we always used 2.69 to generate the script.

Reply:
> (which is 11-year old, BTW)

Yes, it's old, but it's the default version of `autoconf` installed on all of the systems here at LLNL, so changing to 2.71 makes our users here nervous (and rightly so).

Reply:
Yes, it's the default on most of my systems as well (except my laptop), but requiring this as a minium, instead of 2.59 which is 20-year old should not impact that many people (if any).

Reply:
Thanks for the explanation. All this looks good to me. Thanks!

Reply:
CentOS9 [aka RHEL9] has 2.69 - so its likely to live for another 4-9 years?..

Reply:
As long as Intel (or any other vendor) does not screw things up again, there should be no need for another update.